src/test/java/io/axoniq/axonserver/connector/admin/AdminChannelIntegrationTest.java

```diff
@@ -45,7 +45,7 @@
  * @author Sara Pellegrini
  * @since 4.6.0
  */
-class AdminChannelIntegrationTest extends AbstractAxonServerIntegrationTest {
+class AdminChannelIntegrationTest  extends AbstractAxonServerIntegrationTest {
```

Comment:
I am missing a test for the `getBalancingStrategies` method you introduced.

Reply:
It is there already:

```
     CompletableFuture<Void> accepted = adminChannel.getBalancingStrategies()
                .thenCompose(balancingStrategies -> adminChannel.loadBalanceEventProcessor("processor", "tokenStore", balancingStrategies.get(0).getStrategy()));
```